google-cloud-asset/synth.py

```diff
@@ -18,263 +18,22 @@
 import synthtool.gcp as gcp
 import synthtool.languages.ruby as ruby
 import logging
-import os
-import re
-import shutil
-from subprocess import call
 
 logging.basicConfig(level=logging.DEBUG)
 
-gapic = gcp.GAPICGenerator()
-gapic2 = gcp.GAPICMicrogenerator()
-
-v1_library = gapic.ruby_library(
-    'asset', 'v1', artman_output_name='google-cloud-ruby/google-cloud-asset',
-    config_path='artman_cloudasset_v1.yaml'
-)
-s.copy(v1_library / 'lib/google/cloud/asset/v1.rb')
-s.copy(v1_library / 'lib/google/cloud/asset/v1')
-s.copy(v1_library / 'test/google/cloud/asset/v1')
-s.copy(v1_library / 'lib/google/cloud/asset.rb')
-s.copy(v1_library / 'README.md')
-s.copy(v1_library / 'LICENSE')
-s.copy(v1_library / '.gitignore')
-s.copy(v1_library / '.yardopts')
-s.copy(v1_library / 'google-cloud-asset.gemspec', merge=ruby.merge_gemspec)
-
-v1beta1_library = gapic.ruby_library(
-    'asset', 'v1beta1', artman_output_name='google-cloud-ruby/google-cloud-asset',
-    config_path='artman_cloudasset_v1beta1.yaml'
-)
-s.copy(v1beta1_library / 'lib/google/cloud/asset/v1beta1.rb')
-s.copy(v1beta1_library / 'lib/google/cloud/asset/v1beta1')
-s.copy(v1beta1_library / 'test/google/cloud/asset/v1beta1')
-
-orgpolicy_library = gapic2.ruby_library(
-    'orgpolicy', 'v1',
+gapic = gcp.GAPICMicrogenerator()
+library = gapic.ruby_library(
+    "asset", "v1",
     generator_args={
         "ruby-cloud-gem-name": "google-cloud-asset",
+        "ruby-cloud-title": "Cloud Asset",
+        "ruby-cloud-description": "A metadata inventory service that allows you to view, monitor, and analyze all your GCP and Anthos assets across projects and services.",
+        "ruby-cloud-env-prefix": "ASSET",
+        "ruby-cloud-wrapper-of": "v1:0.0;v1beta1:0.0",
```

Comment:
Do you plan to release the v1 and v1beta1 packages, and update `0.0` to `0.1` before merging this PR?

Reply:
I don't think we need to. `~> 0.0` is fine as a dependency.